dashboard-frontend/src/helpers/strings.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { createSlugFor } from './strings';
+
+describe('createSlugFor', () => {
```

Comment:
This is cool. I like the idea of being able to use this during configuration time!